base/doc/fntguide.tex

```diff
@@ -2170,8 +2170,259 @@ \section{Additional text symbols -- \textsf{textcomp}}
+encoding, |acc| indicates a `constructed' accent.
+
+\begin{table}[tbp]
+  \centering\footnotesize
+  \begin{tabular}[t]{@{}lp{1.5em}l@{}}
+    \verb|\textquotestraightbase|     & \textquotestraightbase     & 13  \\
+    \verb|\textquotestraightdblbase|  & \textquotestraightdblbase  & 18  \\
+    \verb|\textcapitalcompwordmark|   & \textcapitalcompwordmark   & 23  \\
+    \verb|\textascendercompwordmark|  & \textascendercompwordmark  & 31  \\
+    \verb|\textdollar|                & \textdollar                & 36  \\
+    \verb|\textquotesingle|           & \textquotesingle           & 39  \\
+    \verb|\textasteriskcentered|      & \textasteriskcentered      & 42  \\
+    \verb|\textdagger|                & \textdagger                & 132 \\
+    \verb|\textdaggerdbl|             & \textdaggerdbl             & 133 \\
+    \verb|\textperthousand|           & \textperthousand           & 135 \\
+    \verb|\textbullet|                & \textbullet                & 136 \\
+    \verb|\texttrademark|             & \texttrademark             & 151 \\
+    \verb|\textcent|                  & \textcent                  & 162 \\
+    \verb|\textsterling|              & \textsterling              & 163 \\
+    \verb|\textyen|                   & \textyen                   & 165 \\
+    \verb|\textbrokenbar|             & \textbrokenbar             & 166
+  \end{tabular}\qquad
+  \begin{tabular}[t]{@{}lp{1.5em}l@{}}
+    \verb|\textsection|               & \textsection               & 167 \\
+    \verb|\textcopyright|             & \textcopyright             & 169 \\
+    \verb|\textordfeminine|           & \textordfeminine           & 170 \\
+    \verb|\textlnot|                  & \textlnot                  & 172 \\
+    \verb|\textregistered|            & \textregistered            & 174 \\
+    \verb|\textdegree|                & \textdegree                & 176 \\
+    \verb|\textpm|                    & \textpm                    & 177 \\
+    \verb|\textparagraph|             & \textparagraph             & 182 \\
+    \verb|\textperiodcentered|        & \textperiodcentered        & 183 \\
+    \verb|\textordmasculine|          & \textordmasculine          & 186 \\
+    \verb|\textonequarter|            & \textonequarter            & 188 \\
+    \verb|\textonehalf|               & \textonehalf               & 189 \\
+    \verb|\textthreequarters|         & \textthreequarters         & 190 \\
+    \verb|\texttimes|                 & \texttimes                 & 214 \\
+    \verb|\textdiv|                   & \textdiv                   & 246
+  \end{tabular}
+  \caption{Default symbols always available in \texttt{TS1} encoding}
+  \label{tab:ts1-subset-always}
+\end{table}
+
+\begin{table}[tbp]
+  \centering
+  \begin{tabular}[t]{@{}lll@{}}
+    \verb*|\textcircled{}|       & \textcircled{}          & acc
+  \end{tabular}
+  \caption{Symbol defined in \texttt{TS1} sub-encoding \texttt{1}}
+  \label{tab:ts1-subset-1}
```

Comment:
The subset number is a bit strange (I know I'm to blame) but saying that something is defined in sub-encoding X is the wrong way around. \textcircled is only in 0 (at level 1 it is undefined). So the caption should probably say

   Symbol only available in sub-encoding 0

or

  Symbol not available in sub-encodings 1--9